apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -238,7 +260,7 @@ export const EventTypeList = ({
       await utils.viewer.eventTypes.getByViewer.cancel();
       const previousValue = utils.viewer.eventTypes.getByViewer.getData();
       if (previousValue) {
-        const newList = [...types];
+        const newList = [...types.map(normalizeEventType)];
```

Comment:
Optimistic updates need to do in memory conversion to make the format consistent with react query response for getByViewer